### DISCUSSION TITLE
how do i train i get the pictures but idk what to do with them i can pay $

### DISCUSSION THREAD

**Discussion by forbid17** [zeyad-mansour/lunar#73] [General]
how do i train i get the pictures but idk what to do with them i can pay $

**Comment by ShotgunSpoon** [comment]
I would liekt o know this as well. What training does itself, how to do it, and how worth the time and gpu power it'll be to do so. I don't want tot rain it if its just for like skins and such, if it will actually improve over all accuracy and ability.
